Source/WebCore/platform/graphics/filters/FEGaussianBlur.cpp

```diff
@@ -140,11 +140,27 @@ bool FEGaussianBlur::resultIsAlphaImage(const FilterImageVector& inputs) const
     return inputs[0]->isAlphaImage();
 }
 
+OptionSet<FilterRenderingMode> FEGaussianBlur::supportedFilterRenderingModes() const
+{
+    OptionSet<FilterRenderingMode> modes = FilterRenderingMode::Software;
+#if USE(GRAPHICS_CONTEXT_FILTERS)
+    if (m_stdX == m_stdY)
```

Comment:
This requires the new platform version-based #ifdef.

Reply:
`#if HAVE(CGSTYLE_COLORMATRIX_BLUR)` is used instead.